IS0501Test.py

```diff
@@ -811,15 +817,15 @@ def check_bulk_stage(self, port, portList):
         data = []
         ports = {}
         for portInst in portList:
-            valid, response = self.is05_utils.generate_destination_ports(port, portInst)
+            valid, response, changeable_param = self.is05_utils.generate_changeable_param(port, portInst)
             if valid:
                 ports[portInst] = response
                 toAdd = {}
                 toAdd['id'] = portInst
                 toAdd['params'] = {}
                 toAdd['params']['transport_params'] = []
                 for portNum in ports[portInst]:
-                    toAdd['params']['transport_params'].append({"destination_port": portNum})
+                    toAdd['params']['transport_params'].append({changeable_param: portNum})

```

Comment:
Minor detail, but could 'portNum' be changed to something else given it's no longer always a port number? Even if it's just 'changeable_value'.